tests/network/vmi_networking.go

```diff
@@ -62,8 +62,6 @@ var _ = SIGDescribe("[Serial][rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com]
 	var currentConfiguration v1.KubeVirtConfiguration
 
 	var inboundVMI *v1.VirtualMachineInstance
-	var inboundVMIWithPodNetworkSet *v1.VirtualMachineInstance
```

Comment:
If the tests are properly isolated now, you can remove the `Serial` tag to allow parallel execution.